pkg/roachprod/install/cockroach.go

```diff
@@ -903,21 +907,25 @@ func (c *SyncedCluster) generateStartArgs(
 
 	// if neither --log nor --log-config-file are present
 	if idx1 == -1 && idx2 == -1 {
-		loggingConfig, err := execLoggingTemplate(loggingTemplateData{
-			LogDir: logDir,
-		})
-		if err != nil {
-			return nil, errors.Wrap(err, "failed rendering logging template")
-		}
+		if !startOpts.EnableFluentSink {
+			args = append(args, "--log", `file-defaults: {dir: '`+logDir+`', exit-on-error: false}`)
```

Comment:
This was the original logic before https://github.com/cockroachdb/cockroach/pull/123227.